src/test/Quorum.test.ts

```diff
@@ -1,7 +1,7 @@
-import { NetworkGraphNode, QuorumSetGroup } from "../Types/NetworkTypes";
+import { NetworkGraphNode, QuorumSet } from "../Types/NetworkTypes";
```

Comment:
This is just fixing this test which was broken in a previous refactor.  I should set github up to run tests automatically